content/02.main-text.md

```diff
@@ -97,6 +97,7 @@ The Deep Review was initiated in August 2016, and the first complete manuscript
 While the article was under review, we continued to maintain the project and accepted new contributions.
 The preprint was updated in January 2018, and the article was accepted by the journal in March 2018 [@doi:10.1098/rsif.2017.0387].
 As of June 15, 2018, the Deep Review repository accumulated {{total_commits}} git commits, {{merged_pull_requests}} merged pull requests, {{open_issues + closed_issues}} issues, and {{github_stars}} GitHub stars.
+[Click here](https://mybinder.org/v2/gh/greenelab/meta-review/master?filepath=analyses/deep-review-contrib/02.contrib-viz.ipynb) to launch Binder and interactively explore alternative visualizations in the Jupyter notebook that generated this figure.
```

Comment:
An alternative phrasing that is not necessarily better than the existing phrasing (so feel free to disregard):

> The notebooks to generate this figure can be [interactively launched using Binder](https://mybinder.org/v2/gh/greenelab/meta-review/master?filepath=analyses/deep-review-contrib/02.contrib-viz.ipynb), enabling users to explore alternative visualizations or analyses of the source data.

Reply:
I switched to that in ac45b94 but used the singular "notebook".  I haven't tested the Python kernels extensively, and we don't provide a direct link to open them.

I also switched to the binder tag and noted it in 7512bcd.